src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -4274,7 +4280,7 @@ void func_8083819C(Player* this, PlayState* play) {
 }
 
 void func_8083821C(Player* this) {
-    s32 i;
+    s32 i = 0;
```

Comment:
```suggestion
    s32 i = 0;
```
curious why add this? it gets initialized in the for loop

Reply:
technically a mistake; I found a way to match the code below using a while loop, eliminating the `clang-format off`, but I reverted it and must have missed this line.